.github/code/customize_configs_for_pipeline.py

```diff
@@ -0,0 +1,56 @@
+import json
+from pathlib import Path
+import argparse
+
+
+def load_json(path: Path) -> dict:
+    with open(path, 'r') as f:
+        return json.load(f)
+
+
+def save_json(data: dict, path: Path):
+    with open(path, 'w') as f:
+        json.dump(data, f, indent=4)
+
+
+def remove_session_placeholders_from_tracker(tracker: dict) -> dict:
+    tracker["PATHS"] = [
+        path.replace("[[NIPOPPY_BIDS_SESSION_ID]]/", "").replace("_[[NIPOPPY_BIDS_SESSION_ID]]", "") 
+        for path in tracker["PATHS"]
+    ]
+    if tracker.get("PARTICIPANT_SESSION_DIR"):
+        tracker["PARTICIPANT_SESSION_DIR"] = tracker["PARTICIPANT_SESSION_DIR"].replace(
+            "/[[NIPOPPY_BIDS_SESSION_ID]]/", ""
+        ).replace("[[NIPOPPY_BIDS_SESSION_ID]]/", "")  # Some Freesurfer trackers have session ID first
+
+    return tracker
```

Comment:
At this point, wouldn't it be easier / more maintanable to have two different templates for each pipeline, one with sessions, and one without? 

And mid-term: we should have a chat with our Nipoppy friends about how such a scenario should be handled. I.e. would they expect there to be two different pipelines in the Zenodo community that are identical in all ways except one is for "has_session" and the other is not